mkdocs/docs/HPC/migration_to_rhel9.md

```diff
@@ -0,0 +1,129 @@
+
+# Migration to RHEL9 operation system (Tier-2)
+
+During the second half of 2024 we will migrate the HPC-UGent Tier-2 clusters that 
+are using RHEL 8 as operating system to **RHEL9 (Red Hat Enterprise Linux 9)**.
+This includes clusters skitty, joltik, doduo, accelgor, donphan and gallade 
+(see also the [infrastructure overview](https://www.ugent.be/hpc/en/infrastructure)), 
+as well as the Tier-2 login nodes.
+
+## Motivation
+
+Migrating to RHEL8 is done to bring all clusters in line with the most recent 
+cluster that are already running RHEL9 (shinx).
+
+This makes the maintenance of the HPC-UGent Tier-2 infrastructure significantly easier, 
+since we only need to take into account a single operating system version going forward.
+
+It will also bring you the latest versions in operating system software, with more 
+features and improved security.
+
+## Impact on the login nodes
+
+As a general rule, your login node should match the operating system of the cluster 
+you are running on. To make this more transparent, you will be prompted when loading 
+a cluster with a different operation system.
+
+    module swap cluster/shinx
+
+    The following have been reloaded with a version change:
+    1) cluster/doduo => cluster/shinx         3) env/software/doduo => env/software/shinx
+    2) env/slurm/doduo => env/slurm/shinx     4) env/vsc/doduo => env/vsc/shinx
+
+    We advise you to log in to a RHEL 9 login node when using the shinx cluster.
+    The banette cluster is using RHEL 9 as operating system,
+    while the login node you are logged in to is using RHEL 8.
+    To avoid problems with testing installed software or submitting jobs,
+    it is recommended to switch to a RHEL 9 login node by running 'ssh login9'.
+
+Initially there will be only one RHEL 9 login node. As needed a second one will be added.
+
+When the default cluster (doduo) migrates to RHEL 9 the corresponding login nodes 
+will also become default (when you do `ssh vsc4xxxx@login.hpc.ugent.be`). 
+When they are nog longer needed the old RHEL 8 login nodes will be shut down.
+
+### Limits
+
+To encourage only using the login nodes as an entry portal to the HPC-UGent infrastructure, 
+user limits will be enforced on the RHEL9 login nodes.
```

Comment:
add `(as was done in on the EL8 loginnodes, but the limits are bit tighter now)`